### ISSUE TITLE
Lots of 'debug' messages in  in '.xsession-errors' file

### ISSUE THREAD

**Issue by ACinnamonUserInUK** [Ulauncher/Ulauncher#99]
It seems to me that my build of Ulauncher has some debugging flag turned on - even though I installed Ulauncher from the standard .deb available on the web.

Here are the messages.

```
2017-03-07 01:23:05,605:DEBUG:ulauncher: main() 'Starting a new instance...'
2017-03-07 01:23:05,706:DEBUG:ulauncher.ui.windows.Builder: do_connect() 'connect builder by name 'ulauncher_window','destroy', 'on_destroy''
2017-03-07 01:23:05,709:DEBUG:ulauncher.ui.windows.Builder: do_connect() 'connect builder by name 'ulauncher_window','focus_in_event', 'on_focus_in_event''
2017-03-07 01:23:05,709:DEBUG:ulauncher.ui.windows.Builder: do_connect() 'connect builder by name 'ulauncher_window','focus_out_event', 'on_focus_out_event''
2017-03-07 01:23:05,711:DEBUG:ulauncher.ui.windows.Builder: do_connect() 'connect builder by name 'input','key_press_event', 'on_input_key_press_event''
2017-03-07 01:23:05,722:DEBUG:ulauncher.ui.windows.Builder: log_unconnected_functions() 'Not connected to builder 'on_mnu_about_activate''
2017-03-07 01:23:05,723:DEBUG:ulauncher.ui.windows.Builder: log_unconnected_functions() 'Not connected to builder 'on_mnu_close_activate''
2017-03-07 01:23:05,723:DEBUG:ulauncher.ui.windows.Builder: log_unconnected_functions() 'Not connected to builder 'on_preferences_dialog_destroyed''
2017-03-07 01:23:05,723:DEBUG:ulauncher.ui.windows.Builder: log_unconnected_functions() 'Not connected to builder 'on_mnu_preferences_activate''
2017-03-07 01:23:05,723:DEBUG:ulauncher.ui.windows.Builder: log_unconnected_functions() 'Not connected to builder 'on_input_changed''
```

OS: Linux Mint 18.1 xenial x86_64 
Kernel: 4.8.0-39-generic 
DE: Cinnamon 
WM: Mutter (Muffin) 
CPU: Intel 2 Duo T7500 (2) @ 2.2GHz 
GPU: Intel Integrated Graphics 

Ulauncher 2.0.13

**Comment by gornostal** [comment]
Thanks for creating this issue. 
I'm going to address this in the next release.

**Comment by gornostal** [comment]
Fixed in v2.1.1 (Dev) by setting default log level to ERROR.
